tests/Basics/HybridLogicalClockTest.cpp

```diff
@@ -39,6 +39,25 @@
 
 using namespace arangodb;
 
+namespace {
+
+class HybridLogicalClockWithFixedTime final
+    : public basics::HybridLogicalClock {
+ public:
+  explicit HybridLogicalClockWithFixedTime(uint64_t t) : _fixed(t) {}
+
+ protected:
+  uint64_t getPhysicalTime() override {
+    // arbitrary timestamp from Sep 30, 2022.
```

Comment:
It is not clear where this date is set (I believe there's a timestamp set when the constructor is called later, maybe this comment is better placed there?)

Reply:
Thanks for the review.
I added code comments to the test and added another test to the PR. Now it is done from my POV.